bin/docker/kafka_bridge_tls_prepare_certificates.sh

```diff
@@ -27,43 +27,45 @@ function create_keystore {
    RANDFILE=/tmp/.rnd openssl pkcs12 -export -in $3 -inkey $4 -name $5 -password pass:$2 -out $1 -certpbe aes-128-cbc -keypbe aes-128-cbc -macalg sha256
 }
 
-# $1 = trusted certs, $2 = TLS auth cert, $3 = TLS auth key, $4 = truststore path, $5 = keystore path, $6 = certs and key path
-trusted_certs=$1
-tls_auth_cert=$2
-tls_auth_key=$3
-truststore_path=$4
-keystore_path=$5
-certs_key_path=$6
+# Parameters:
+# $1: Path to the new truststore
+# $2: Truststore password
+# $3: Base path where the certificates are mounted
+# $4: Environment variable defining the certs that should be loaded
+function prepare_truststore {
+    STORE=$1
+    PASSWORD=$2
+    BASEPATH=$3
+    TRUSTED_CERTS=$4
```

Comment:
See comment https://github.com/strimzi/strimzi-kafka-operator/pull/10123#discussion_r1607982686 same applies here.

Reply:
I renamed the `STORE` here as well as in the other PR to `TRUSTSTORE`. Please let me know if it works for you and if it is more consistent.